src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -14,23 +15,28 @@ internal class CoreXT
     {
         public string ConfigFilePath { get; }
         public string ComponentsFilePath { get; }
-        public XDocument Config { get; }
-        public JObject Components { get; }
+        public XDocument Config { get; }        
+        private static Dictionary<string, string> ComponentToFileMap { get; set; }
+        private static Dictionary<string, JObject> ComponentFileToDocumentMap { get; set; }
```

Comment:
Why private property?

Reply:
There is no requirement for consumer of this class to get hold of these maps. Hence private. But yes there is no need to make them properties. Will remove that.